frontend/src/i18n.ts

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+i18n
```

Comment:
Is this file mandatory , or was it created for our own purposes? I am asking because it is used directly in a few places, and the name could lead to confusion.
I noticed a few places where it is used, and it can also be used for the missing translation points 1-4. Or using Translation (https://react.i18next.com/latest/translation-render-prop)